source/administration/extended-support-release.rst

```diff
@@ -44,7 +44,7 @@ What are the current supported Extended Support Release versions?
 +-------------+----------------+------------------+------------------+--------------------------------------------------------------------------------------------+-------------------------------------------------+
 | Version     | Type           | Release Date     | End of Support   | Latest Dot Release Download link                                                           | Upgrade Notes                                   |
 +=============+================+==================+==================+============================================================================================+=================================================+
-| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.4 <https://releases.mattermost.com/5.25.4/mattermost-5.25.4-linux-amd64.tar.gz>`_    |                                                 |
+| 5.25        | Quality        | July 16, 2020    | April 15, 2021   | `5.25.5 <https://releases.mattermost.com/5.25.5/mattermost-5.25.5-linux-amd64.tar.gz>`_    |                                                 |
```

Comment:
Should we call out upgrade notes here, given this is effectively a breaking change?

Reply:
Thanks @lieut-data, upgrade notes slipped off my radar. Added a note on this page, and also added the Upgrade Notes doc to this PR. Followed @wiersgallak's suggestion in-channel to add a note on that page to all versions after v5.20.

Re-requesting review, if you wouldn't mind taking a look at the upgrade notes as well?